bundle/src/main/java/com/adobe/acs/commons/cloudservices/pwa/impl/ConfigurationImpl.java

```diff
@@ -12,54 +11,56 @@
 import org.apache.sling.models.annotations.Model;
 import org.apache.sling.models.annotations.injectorspecific.OSGiService;
 import org.apache.sling.models.annotations.injectorspecific.Self;
-import org.apache.sling.models.annotations.injectorspecific.SlingObject;
 import org.apache.sling.settings.SlingSettingsService;
 
 import javax.annotation.Nullable;
 import javax.annotation.PostConstruct;
+import java.util.Calendar;
+import java.util.GregorianCalendar;
 
 @Model(
         adaptables = {SlingHttpServletRequest.class},
         adapters = {Configuration.class}
 )
 public class ConfigurationImpl implements Configuration {
-    private static final String RESOURCE_TYPE = "acs-commons/cloudservices/pwa/components/conf/page";
+    //private static final String RESOURCE_TYPE = "acs-commons/cloudservices/pwa/components/conf/page";
     private static final String PN_SERVICE_WORKER_JS_CLIENT_LIBRARY_CATEGORY = "serviceWorkerJsClientLibraryCategory";
     private static final String PN_PWA_JS_CLIENT_LIBRARY_CATEGORY = "pwaJsClientLibraryCategory";
 
     @Self
     private SlingHttpServletRequest request;
 
-
     @OSGiService
     private SlingSettingsService slingSettingsService;
 
-    // Do not use the @SlingObject for this since we want to get it from the Request so we can get the service RR.
-    private ResourceResolver resourceResolver;
-
-    private Page currentPage;
-    private PageManager pageManager;
-
     // These are resolved in init()
+    private ResourceResolver resourceResolver;
     private Page rootPage;
     private Page confPage;
 
     @PostConstruct
     protected void init() {
-
         resourceResolver = request.getResourceResolver();
 
-        pageManager = resourceResolver.adaptTo(PageManager.class);
-        currentPage = pageManager.getContainingPage(request.getResource());
+        final PageManager pageManager = resourceResolver.adaptTo(PageManager.class);
+        final Page currentPage = pageManager.getContainingPage(request.getResource());
 
         Page page = currentPage;
 
         while (page != null) {
             final String confPath = page.getProperties().get("cq:conf", String.class);
 
             if (StringUtils.isNotBlank(confPath)) {
-                Resource cloudConfigsResource = resourceResolver.getResource(confPath + "/settings/cloudconfigs");
-                if (cloudConfigsResource != null) {
+                final Resource jcrContentResource = resourceResolver.getResource(confPath + "/settings/cloudconfigs/pwa/jcr:content");
+
+                if (jcrContentResource != null) {
+                    confPage = pageManager.getContainingPage(jcrContentResource);
+                    rootPage = page;
```

Comment:
rootPage doesn't depend upon the serviceResolver. Hence for an anonymous resolver that's all required.